src/binary/binaryFetcher/binaryValidator.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from "fs";
+import { runProcess } from "../runProcess";
+// import { runProcess } from "../runProcess";
+
+export default async function isValidBinary(version: string): Promise<boolean> {
+  const exists = fs.existsSync(version);
```

Comment:
why sync?

Reply:
oh, this was the original code, ill change it